test/units/modules/network/eos/test_eos_static_routes.py

```diff
@@ -0,0 +1,206 @@
+#
+# (c) 2019, Ansible by Red Hat, inc
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+#
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+from units.compat.mock import patch
+from ansible.modules.network.eos import eos_static_routes
+from units.modules.utils import set_module_args
+from .eos_module import TestEosModule, load_fixture
+
+
+class TestEosStaticRoutesModule(TestEosModule):
```

Comment:
Need to add unit test cases to assert for before and after keys as well

Reply:
will work this as a separate PR